README.md

```diff
@@ -27,11 +27,49 @@ TODO: Write documentation for how to build initrd locally
 
 See [Hacking](HACKING.md)
 
-# Integrating with Kernel Snap
+# Releasing
 
-When your initrd DEB package is built, you need to perform following steps to make it a part of kernel snap:
+The UC initrd is included in kernel snaps. However, the process to get
+it inside the kernel snap is not immediate and consists of a few
+steps. First, we need to build the `ubuntu-core-initramfs`
+debian package in the [snappy-dev/image
+PPA](https://launchpad.net/~snappy-dev/+archive/ubuntu/image) by
+following these steps:
 
-TODO: Write documentation for how to integrate initrd.img to your (custom or canonical) kernel 
+1. Update the changelog with latest changes since last release (use `dch -i` for this)
+1. Propose a PR to the repo with the new changelog, get it reviewed and merged
+1. Tag the repository with the new version
+1. Build the source package by running
+
+        git clean -fdx
+        dpkg-buildpackage -S -sa -d
```

Comment:
This should probably converted to "git buildpackage" at some point, it will ensure the "git clean" step is not needed and that it's always build in a clean tree.

Reply:
Changed to `gbp buildpackage` (you meant that one, right?). But it looks that cleaning up is still needed, although gbp now errors if the tree is not clean, which is nice.